### PR TITLE
Update katello package dependencies

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -95,16 +91,14 @@ Obsoletes: %{?scl_prefix}rubygem-bastion
...
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 3.3.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 3.8.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 4.0.0
 BuildRequires: %{?scl_prefix}npm(babel-loader) >= 8.0.0
 BuildRequires: %{?scl_prefix}npm(babel-loader) < 9.0.0
 BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) >= 3.0.0
 BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) < 4.0.0
 # end package.json devDependencies BuildRequires
 # start package.json dependencies BuildRequires
-BuildRequires: %{?scl_prefix}npm(@theforeman/vendor) >= 3.3.2
```
Comment (ekohl) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: This is now a peer dependency. Should probably be translated as well.

Comment (evgeni) [reply packages/katello/rubygem-katello/rubygem-katello.spec]: Yeah, it won't built without.

Comment (ekohl) [reply packages/katello/rubygem-katello/rubygem-katello.spec]: I suspect that it's pulled in via foreman-assets anyway.

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 2]
```diff
@@ -95,16 +91,14 @@ Obsoletes: %{?scl_prefix}rubygem-bastion
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 3.3.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 3.8.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 4.0.0
```
Comment (ekohl) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: This is an issue, since we have 4.x in RPMs. Will need an upstream PR as well.

Comment (tbrisker) [reply packages/katello/rubygem-katello/rubygem-katello.spec]: cc @sharvit

**Comment by ehelms** [comment]
Included https://github.com/theforeman/foreman-packaging/pull/4569

**Pull request closed without merging**